File: src/main/java/no/nav/pto/veilarbportefolje/opensearch/domene/Endring.java
package no.nav.pto.veilarbportefolje.opensearch.domene;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class Endring {
    public String aktivtetId;
    public String tidspunkt;
    public String er_sett;
}

File: src/main/java/no/nav/pto/veilarbportefolje/opensearch/domene/Endring.kt
package no.nav.pto.veilarbportefolje.opensearch.domene

import lombok.Data
import lombok.experimental.Accessors

@Data
@Accessors(chain = true)
data class Endring(
    var aktivtetId: String? = null,
    var tidspunkt: String? = null,
    var erSett: String? = null
)
